git_machete/docs.py

```diff
@@ -140,6 +141,25 @@
           <b>-b, --branch=<branch></b>      Branch to set the annotation for.
           <b>-H, --sync-github-prs</b>      Annotate with GitHub PR numbers and authors where applicable.
     """,
+    "clean": """
+        <b>Usage:
+          git machete clean [-c|--checkout-my-github-prs]
+
+        If invoked without any flag, deletes untracked and unmanaged branches.
+
+        If invoked with ``-c`` or ``--checkout-my-github-prs``, checkouts your open PRs into local branches.
+
+        To allow GitHub API access for private repositories (and also to perform side-effecting actions like opening a PR, even in case of public repositories),
+        a GitHub API token with ``repo`` scope is required, see https://github.com/settings/tokens. This will be resolved from the first of:
+
+            1. ``GITHUB_TOKEN`` env var,
```

Comment:
Hmmm where's that piece taken from? There's already the 4th option as well (`~/.github-token`), so it's missing both here and probably in the original place as well!

Reply:
Its from `anno` command

Reply:
Ok pls update both

Reply:
Ok pls update both